hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -1682,14 +1686,15 @@
       name: chip_sw_hmac_idle
       desc: '''Verify the HMAC clk idle signal to clkmgr.
 
-            - Write the HMAC clk hint to 1 within clkmgr to indicate HMAC clk is ready to be gated.
-              Verify that the HMAC clk hint status within clkmgr reads 0 (HMAC is idle).
+            - Write the HMAC clk hint to 0 within clkmgr to indicate HMAC clk can be gated and
+              verify that the HMAC clk hint status within clkmgr reads 0 (HMAC is disabled).
```

Comment:
This PR is great! Thanks for adding that.
Just a quick comment - I guess we do not have a way to really prove that HMAC is off expect reading clock status. Like how we ensure HMAC is on via doing a transaction.